src/styles/StyleUtils.js

```diff
@@ -319,6 +319,26 @@ function getModalPaddingStyles({
     };
 }
 
+/**
+ * Gives the margin style for login container
+ *
+ * @param {Object} params
+ * @returns {Object}
+ */
+function getLoginContainerMarginStyles({
+    isSmallScreenWidth,
+    isMediumScreenWidth,
+    windowHeight,
+}) {
+    if (isSmallScreenWidth) {
+        return styles.signInPageNarrowContentMargin;
+    }
+    if (!isMediumScreenWidth || (isMediumScreenWidth && windowHeight < variables.minHeightToShowGraphics)) {
+        return styles.signInPageWideLeftContentMargin;
+    }
+    return {};
+}
```

Comment:
This seems like an error can you recheck it, please?

For `isSmallScreenWidth` && `styles.signInPageWideLeftContentMargin` also should apply because `!isMediumScreenWidth` can be `isSmallScreenWidth = true` right?

Reply:
Ohh You are absolutely right. I'm editing it as a spread operator in case other style types may come in the future.
If you disagree with this opinion, I can use the assignment operator right away. 
Thank you so much for your warning of my mistake 🙏 .

Reply:
Can you share both snippets for better understanding, thanks!

Reply:
version 1 :
```
let marginStyle = {};
    if (isSmallScreenWidth) {
        marginStyle = {...marginStyle, ...styles.signInPageNarrowContentMargin};
    }
    if (!isMediumScreenWidth || (isMediumScreenWidth && windowHeight < variables.minHeightToShowGraphics)) {
        marginStyle = {...marginStyle, ...styles.signInPageWideLeftContentMargin};
    }
    return marginStyle;
```
version 2:
```
let marginStyle = {};
    if (isSmallScreenWidth) {
        marginStyle = styles.signInPageNarrowContentMargin;
    }
    if (!isMediumScreenWidth || (isMediumScreenWidth && windowHeight < variables.minHeightToShowGraphics)) {
        marginStyle = styles.signInPageWideLeftContentMargin;
    }
    return marginStyle;
```

If there is a possibility of additional style information, I think the 1st method may be more useful in the future.

Reply:
Version 2 is still invalid

Reply:
I'm not writing to say that version 2 is beautiful. Version 2 also works because it has one variable. I think version 1 is more consistent for the future, of course.

```
    signInPageNarrowContentMargin: {
        marginTop: '40%',
    },
    signInPageWideLeftContentMargin: {
        marginTop: '44.5%',
    },
```

Reply:
There seems to be a problem with the resolution(883 x 478). I'll try it and report back.  The method I mentioned in my proposal seems to cause something else.